src/main/java/Main.java

```diff
@@ -0,0 +1,5 @@
+public class Main {
+    public static void main(String[] args) {
+
```

Comment:
you should test your code here, its important

Reply:
see main() method strucrture

![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/8808fd7c-c9e4-4665-b8f9-77719fdf78a7)